xdmod/README.md

```diff
@@ -62,39 +68,222 @@ The following information is needed by Open XDMoD:
+
+#### What about a Heterogeneous cluster?
+
+The `xdmod-setup` script is used for the basic setup of Open XDMoD. The script includes options to configure the Open XDMoD database, setup the admin user account and configure resources.
+Open XDMoD's [Configuration](https://open.xdmod.org/configuration.html#location-of-configuration-files) files.
+
+
+*Have a heterogeneous cluster?*  You would need to modify `/etc/xdmod/resource_specs.json`:
+
+[![asciicast](https://asciinema.org/a/349249.svg)](https://asciinema.org/349249)
+
+
+#### Hierarchy
+
+Open XDMoD supports a three level hierarchy.
+In this tutorial we use a hierarchy configuration that is typical of the organizational structure in a University.
+
+Decanal Unit -> Department -> PI Group
+
+Reference: [Hierarchy Guide](https://open.xdmod.org/hierarchy.html)
+
+#### User / PI Names
+
+The resource manager logs contain the system usernames of the users that submitted jobs.
+To display the full names in Open XDMoD you must provide a data file that contains the 
+full name of each user for each system username. This file is in a `csv` format.
+
+![Group By User(names not importe)](./tutorial-screenshots/usernames.png)
+
+This has not been automated for this tutorial. We dont want you to fall asleep!
+
+Create a file with the contents below:
+
+The file needs to be able to be read by the `xdmod` user, for this demo it will be
+created in /var/tmp
+
+```bash
+vim /var/tmp/names.csv
+```
+
+The first column should include the user name or group name used by your resource manager, the second column is the user’s first name and the third column is the user’s last name.
+(Feel free to change the First and Last names)
+
+```csv
+cgray,Carl,Gray
+sfoster,Stephanie,Foster
+csimmons,Charles,Simmons
+astewart,Andrea,Stewart
+hpcadmin,HPC, Administrators
+```
+
+Now this needs to be imported into xdmod with the command [`xdmod-import-csv`](https://open.xdmod.org/commands.html#xdmod-import-csv)
+
+```bash
+sudo su - xdmod xdmod-import-csv -t names -i /var/tmp/names.csv
+```
+
+
+![Group By User](./tutorial-screenshots/fullnames.png)
+
+Reference: [User/PI Names Guide](https://open.xdmod.org/user-names.html)
+
+xdmod-import-csv -t names:
+[![asciicast](https://asciinema.org/a/349325.svg)](https://asciinema.org/349325)
+
 
 
 ## Open XDMoD Job Performance
 The Job Performance module is optional, but highly recommended.
-The Job Performance
-TODO: PCP Configuration (mention alternatives, TACC,...)
-https://github.com/ubccr/hpc-toolset-tutorial/blob/master/slurm/pmlogger-supremm.config#L56-L59
 
-Done: Names.csv
+![Job Performance Dataflow](./tutorial-screenshots/admin-job-performance-dataflow.png)
+
+### Job Performance Configuration
+
+This tutorial uses the [Job Performance](https://supremm.xdmod.org) module for Open XDMoD. The Job Performance module displays job performance metric based on data collected on the HPC compute nodes.
```

Comment:
Job Performance data  - for the Open source release we'll try to provide support for PCP. We chose PCP because it is included by default in Centos / RedHat.

In XSEDE we use tacc_stats and PCP (depending on the resource provider). and we have also used LDMS, Cray RUR and are award of groups using Ganglia too.